ceph.go

```diff
@@ -749,9 +750,20 @@ func parseCephCommand(out string, err error, v interface{}) error {
 		return err
 	}
 
-	if err := json.Unmarshal([]byte(out), v); err != nil {
+	sanitized := handleCephInf([]byte(out))
+	if err := json.Unmarshal(sanitized, v); err != nil {
 		return err
 	}
 
 	return nil
 }
+
+// Some Ceph commands can return "inf" as a float value; this is not allowed by
+// the json spec or the golang parser (though it is apparently allowed by the
+// Python parser), so we convert such cases to "null".
```

Comment:
Just FYI: I used Python for validating the Ceph JSON output to find out why pgremapper was throwing errors. It did what I expected it to so, so it's definitely not allowed.

Apparently, what Python does is replace `inf` with JavaScript's `Infinity` (same for `NaN`):
```python
>>> json.dumps({'x': float('inf')})
'{"x": Infinity}'
>>> json.loads(json.dumps({'x': float('inf')}))
{'x': inf}
```

Reply:
Ah, interesting. I had done some digging when I wrote this for ceph_exporter and thought I had found references to Python accepting this, but I'd have to go digging for those references again. Perhaps I had also assumed that the mgr handled this OK without custom code.